### PR TITLE
change GriffinCamp to LaCommune

### PR DESCRIPTION
**Pull request by noratop**
I changed DecodeMTL location in the footer, from GriffinCamp to LaCommune.

### CODE REVIEW

File: src/_data/faq.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     },
     {
       "question": "Where is the course taught?",
-      "answer": "These courses are currently taught at the GriffinCamp (1384 Rue Notre-Dame West, Montréal, QC H3C 1K8) in the heart of Montreal's innovation district."
+      "answer": "These courses are currently taught at La Commune (440 Place Jacques Cartier, 3rd floor, H2Y 2B3) in the heart of Montreal's innovation district."
     }
```
Comment (ziad-saab) [inline src/_data/faq.json:38]: it's actually on the 4th floor ;)

File: src/_data/fulltime_faq.json [hunk 2]
```diff
@@ -46,7 +46,7 @@
     },
     {
       "question": "Where is the course taught?",
-      "answer": "These courses are currently taught at the GriffinCamp (1384 Rue Notre-Dame West, Montréal, QC H3C 1K8) in the heart of Montreal's innovation district."
+      "answer": "These courses are currently taught at the La Commune (440 Place Jacques Cartier, 3rd floor, H2Y 2B3) in the heart of Montreal's innovation district."
     }
```
Comment (ziad-saab) [inline src/_data/fulltime_faq.json:50]: 4th floor :)

**Comment by ziad-saab** [comment]
This is great. Thank you for being thorough and changing Griffin Camp in all locations :) Fix these two little things and we can merge.

**Comment by noratop** [comment]
Floor indication at La Commune for the course location has been updated.

**Please review _La Commune_ description that I suggested for the "About DecodeMTL" page.**

**Comment by ziad-saab** [comment]
Excellent. If this were a real pull request, I would be merging it now :)

**Pull request closed without merging**